src/services/getPreparedTodos.ts

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+
+export function getPreparedTodos(
+  todos: Todo[],
+  { query, status }: { query: string; status: string },
```

Comment:
The `status` parameter is expected to be a string, but it might be beneficial to define a type or enum for it to ensure only valid statuses ('active', 'completed', etc.) are used. This can help prevent errors from invalid status values.